docs/kh2/file/type/00battle.md

```diff
@@ -48,30 +48,36 @@ The damage effects' values on MSET files point to this table.
 |--------|---------------|-------------|
 | 0 	 | ushort | SubId
 | 2 	 | ushort | Id
-| 4 	 | byte | Pierce (0 normal, 1 pierces armor)
-| 5 	 | byte | Damage reduction (0 normal, 01 half damage, 2 no damage) (Needs confirmation)
+| 4 	 | byte | Type (0 normal, 1 pierces armor...)
+| 5 	 | byte | Critical Adjust (0 normal, 1 half damage, 2 no damage)
 | 6 	 | ushort | Power
-| 8 	 | byte | Target (0/1/2 Enemies, 3/4/5 Enemies and allies) (Needs confirmation)
+| 8 	 | byte | Team (Deal damage to: 0/1/2 Enemies, 3/4/5 Enemies and allies...)
 | 9 	 | byte | Element (0 phys, 1 fire, 2 blizz, 3 thun...)
-| 10 	 | byte | Knockback Type (Check below)
+| 10 	 | byte | Reaction (Wether an enemy is flinched, knocked...)
 | 11 	 | byte | Effect on hit (0 none, other values = different effects)
 | 12 	 | short | Knockback Strength 1 (Distance depends on enemy weight)
 | 14 	 | short | Knockback Strength 2 (Distance depends on enemy weight)
 | 16 	 | short | ???
-| 18 	 | byte | Attack type (Eg: 20/22 can defeat bosses)
-| 19 	 | short | ???
+| 18 	 | byte | Flag (Eg: 20/22 can defeat bosses)
+| 19 	 | byte | Refact Self
```

Comment:
If this "Refact" or "Reflect"?

Reply:
Refact, no idea what it comes from, taken from code.